boa/src/builtins/math/mod.rs

```diff
@@ -502,11 +615,29 @@ impl Math {
     /// [spec]: https://tc39.es/ecma262/#sec-math.max
     /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max
     pub(crate) fn max(_: &JsValue, args: &[JsValue], context: &mut Context) -> JsResult<JsValue> {
+        // 1. Let coerced be a new empty List.
+        // 2. For each element arg of args, do
+        // b. Append n to coerced.
+        // 3. Let highest be -∞𝔽.
         let mut max = f64::NEG_INFINITY;
```

Comment:
Id say calling it `highest` or similar like the spec would be nicer to avoid `max.max(num)`

Reply:
Yeah, also the `min` in `Math.min`